assets/prototype/domain/eventEditor/containers/mutations/useCreateDateMutation.js

```diff
@@ -87,6 +91,9 @@ const useCreateDateMutation = ({ eventId }) => {
 						},
 					},
 				});
+
+				// update relations
+				setRelation('datetimes', datetime.id, 'tickets', tickets);
```

Comment:
I am thinking if it would be more readable to have as an argument an object with keys:
`objectType, objectId, relatedType, relatedIds`, this way we will have more context about how each piece is mapped.